css/portal-dashboard/all-responses-popup/selected-student-list.less

```
@@ -0,0 +1,198 @@
+@import "../variables";
+
+.spotlightListDialog {
+  border-radius: 8px;
+  box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.35);
+  background-color: white;
+  position: absolute;
+  top: 25px;
+  left: 25px;
+  right: 25px;
+  bottom: 25px;
+  z-index: 1000;
+  box-shadow: 0 0 0 8000px rgba(0,0,0,0.35);
+
+  .spotlightListHeader {
+    height: 50px;
+    border-radius: 8px 8px 0 0;
+    background-color: @cc-teal-light4;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+    .headerLeft {
+      margin-left: 20px;
+
+      .spotlightIcon {
+        width: 32px;
+        height: 32px;
+        object-fit: contain;
+        fill: @cc-teal;
+      }
+    }
+
+    .headerCenter {
+      display: flex;
+      flex-direction: row;
+      flex: 1;
+      align-items: center;
+      width: 400px;
+      height: 100%;
+      font-size: 16px;
+
+      .assignmentContainer {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        margin: auto;
+
+        .assignmentIcon {
+          fill: @cc-teal;
+        }
+
+        .title {
+          flex: 1;
+          margin-left: 5px;
+          font-weight: bold;
+        }
+      }
+    }
+
+    .headerRight {
+      margin-right: 20px;
+      .closeIcon {
+        width: 27px;
+        height: 27px;
+        background-color: @cc-teal-light6;
+        border-radius: 50%;
+        cursor: pointer;
+        &:hover {
+          background-color: @cc-teal-light3;
+        }
+        &:active {
+          box-sizing: border-box;
+          border: 1.5px solid white;
+          background-color: @cc-teal;
+        }
+        .closeIconSVG {
+          position: relative;
+          top: -1.75px;
+          left: -2.25px;
+          width: 32px;
+          height: 32px;
+          fill: @cc-teal;
+          &:active {
+            position: relative;
+            top: -3.5px;
+            left: -3.75px;
+            fill: white;
+          }
+        }
+      }
+    }
+  }
+
+  .spotlightColumnHeaders {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    font-weight: normal;
+    height: 50px;
+
+    .anonymizeContainter {
+      width: 270px;
+      margin-right: 3px;
+      background-color: @cc-teal-light6;
+      height: 100%;
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      flex-shrink: 0;
+    }
+    .questionPrompt {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      width: 100%;
+      background-color: @cc-teal-light6;
+      height: 100%;
+
+      .icon {
+        fill: @cc-teal;
+        margin-left: 15px;
+      }
+      .questionPromptText {
+        margin-left: 3px;
+        margin-right: 20px;
+        text-overflow: ellipsis;
+        overflow: hidden;
+        white-space: nowrap;
+      }
+    }
+  }
+
+  .selecteStudentResponseTable {
```

```suggestion
  .selectedStudentResponseTable {
```
or perhaps `selectStudentResponseTable`